test/integration/spa.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <meta charset='utf-8'>
+  <title>SDK SPA Test</title>
+  <script src='sdk.js'></script>
+  <script>
+    const clientPort = 3000;
+    const redirectPort = 4000;
+    const redirectUri = `http://localhost:${redirectPort}`;
+    // this file tests a redirect scheme with a separate front end client & back
+    // end server. This is the same scheme that will be used if a developer
+    // chooses to host their redirect on Smartcar's CDN. For the purposes of
+    // this test we hit a local redirect server where we provide the
+    // `app_origin` param ensuring it will post back to that `app_origin` param
+    const options = {
+      // required
+      clientId: 'clientId',
+      // checked against event origin for security
+      redirectUri,
```

Comment:
doesn't this need to be the full url?

Reply:
gotcha, it took me a couple of minutes to fully understand what was going on here hahah

Reply:
lol having written it before Japan it took me a couple minutes to understand too so definitely think the clarification is helpful